--- work/lightsaber/scheme/__asm_pack_unpack.S
+++ work-patched/lightsaber/scheme/__asm_pack_unpack.S
@@ -2,8 +2,8 @@
 .align 2
 .global __asm_1_to_16
 .global ___asm_1_to_16
-__asm_1_to_16:
-___asm_1_to_16:
+PQCLEAN_NAMESPACE__asm_1_to_16:
+_PQCLEAN_NAMESPACE__asm_1_to_16:
 
     mov x15, #8
     _1_to_16_outer_loop:
@@ -44,8 +44,8 @@
 .align 2
 .global __asm_4_to_16
 .global ___asm_4_to_16
-__asm_4_to_16:
-___asm_4_to_16:
+PQCLEAN_NAMESPACE__asm_4_to_16:
+_PQCLEAN_NAMESPACE__asm_4_to_16:
 
     mov x7, #32
     _4_to_16_loop:
@@ -79,8 +79,8 @@
 .align 2
 .global __asm_10_to_32
 .global ___asm_10_to_32
-__asm_10_to_32:
-___asm_10_to_32:
+PQCLEAN_NAMESPACE__asm_10_to_32:
+_PQCLEAN_NAMESPACE__asm_10_to_32:
 
     mov x7, #16
     _10_to_32_loop:
@@ -152,8 +152,8 @@
 .align 2
 .global __asm_13_to_32
 .global ___asm_13_to_32
-__asm_13_to_32:
-___asm_13_to_32:
+PQCLEAN_NAMESPACE__asm_13_to_32:
+_PQCLEAN_NAMESPACE__asm_13_to_32:
 
     mov x7, #8
     _13_to_32_loop:
@@ -306,8 +306,8 @@
 .align 2
 .global __asm_16_to_32
 .global ___asm_16_to_32
-__asm_16_to_32:
-___asm_16_to_32:
+PQCLEAN_NAMESPACE__asm_16_to_32:
+_PQCLEAN_NAMESPACE__asm_16_to_32:
 
     mov x7, #128
     _sbfx_loop:

