--- work/dilithium3/scheme/__asm_iNTT.S
+++ work-patched/dilithium3/scheme/__asm_iNTT.S
@@ -25,13 +25,13 @@
  * SOFTWARE.
  */
 
-#include "macros.i"
+#include "macros.inc"
 
 .align 2
-.global __asm_intt_SIMD_top
-.global ___asm_intt_SIMD_top
-__asm_intt_SIMD_top:
-___asm_intt_SIMD_top:
+.global PQCLEAN_NAMESPACE__asm_intt_SIMD_top
+.global _PQCLEAN_NAMESPACE__asm_intt_SIMD_top
+PQCLEAN_NAMESPACE__asm_intt_SIMD_top:
+_PQCLEAN_NAMESPACE__asm_intt_SIMD_top:
 
     push_all
     Q         .req w20
@@ -435,10 +435,10 @@
     br lr
 
 .align 2
-.global __asm_intt_SIMD_bot
-.global ___asm_intt_SIMD_bot
-__asm_intt_SIMD_bot:
-___asm_intt_SIMD_bot:
+.global PQCLEAN_NAMESPACE__asm_intt_SIMD_bot
+.global _PQCLEAN_NAMESPACE__asm_intt_SIMD_bot
+PQCLEAN_NAMESPACE__asm_intt_SIMD_bot:
+_PQCLEAN_NAMESPACE__asm_intt_SIMD_bot:
 
     push_all
     Q         .req w20

