--- work/dilithium3/scheme/__asm_poly.S
+++ work-patched/dilithium3/scheme/__asm_poly.S
@@ -5,8 +5,8 @@
 .align 2
 .global __asm_10_to_32
 .global ___asm_10_to_32
-__asm_10_to_32:
-___asm_10_to_32:
+PQCLEAN_NAMESPACE__asm_10_to_32:
+_PQCLEAN_NAMESPACE__asm_10_to_32:
 
     mov x7, #16
     _10_to_32_loop:
@@ -78,8 +78,8 @@
 .align 2
 .global __asm_poly_reduce
 .global ___asm_poly_reduce
-__asm_poly_reduce:
-___asm_poly_reduce:
+PQCLEAN_NAMESPACE__asm_poly_reduce:
+_PQCLEAN_NAMESPACE__asm_poly_reduce:
 
     ldr w4, [x1]
 
@@ -171,8 +171,8 @@
 .align 2
 .global __asm_poly_caddq
 .global ___asm_poly_caddq
-__asm_poly_caddq:
-___asm_poly_caddq:
+PQCLEAN_NAMESPACE__asm_poly_caddq:
+_PQCLEAN_NAMESPACE__asm_poly_caddq:
 
     ldr w4, [x1]
 
@@ -264,8 +264,8 @@
 .align 2
 .global __asm_poly_freeze
 .global ___asm_poly_freeze
-__asm_poly_freeze:
-___asm_poly_freeze:
+PQCLEAN_NAMESPACE__asm_poly_freeze:
+_PQCLEAN_NAMESPACE__asm_poly_freeze:
 
     ldr w4, [x1]
 
@@ -393,8 +393,8 @@
 .align 2
 .global __asm_poly_power2round
 .global ___asm_poly_power2round
-__asm_poly_power2round:
-___asm_poly_power2round:
+PQCLEAN_NAMESPACE__asm_poly_power2round:
+_PQCLEAN_NAMESPACE__asm_poly_power2round:
 
     mov w4, #1
 
@@ -539,8 +539,8 @@
 .align 2
 .global __asm_poly_add
 .global ___asm_poly_add
-__asm_poly_add:
-___asm_poly_add:
+PQCLEAN_NAMESPACE__asm_poly_add:
+_PQCLEAN_NAMESPACE__asm_poly_add:
 
     ld1 {v0.4S}, [x1], #16
     ld1 {v4.4S}, [x2], #16
@@ -588,8 +588,8 @@
 .align 2
 .global __asm_poly_sub
 .global ___asm_poly_sub
-__asm_poly_sub:
-___asm_poly_sub:
+PQCLEAN_NAMESPACE__asm_poly_sub:
+_PQCLEAN_NAMESPACE__asm_poly_sub:
 
     ld1 {v0.4S}, [x1], #16
     ld1 {v4.4S}, [x2], #16
@@ -637,8 +637,8 @@
 .align 2
 .global __asm_poly_shiftl
 .global ___asm_poly_shiftl
-__asm_poly_shiftl:
-___asm_poly_shiftl:
+PQCLEAN_NAMESPACE__asm_poly_shiftl:
+_PQCLEAN_NAMESPACE__asm_poly_shiftl:
 
     add x1, x0, #0
 
@@ -704,8 +704,8 @@
 .align 2
 .global __asm_poly_pointwise_montgomery
 .global ___asm_poly_pointwise_montgomery
-__asm_poly_pointwise_montgomery:
-___asm_poly_pointwise_montgomery:
+PQCLEAN_NAMESPACE__asm_poly_pointwise_montgomery:
+_PQCLEAN_NAMESPACE__asm_poly_pointwise_montgomery:
 
     push_all
 
@@ -823,8 +823,8 @@
 .align 2
 .global __asm_polyvecl_pointwise_acc_montgomery
 .global ___asm_polyvecl_pointwise_acc_montgomery
-__asm_polyvecl_pointwise_acc_montgomery:
-___asm_polyvecl_pointwise_acc_montgomery:
+PQCLEAN_NAMESPACE__asm_polyvecl_pointwise_acc_montgomery:
+_PQCLEAN_NAMESPACE__asm_polyvecl_pointwise_acc_montgomery:
 
     push_all
 

