--- work/dilithium3/scheme/ntt.h
+++ work-patched/dilithium3/scheme/ntt.h
@@ -32,26 +32,29 @@
  * SOFTWARE.
  */
 
-#include <stdint.h>
 #include "NTT_params.h"
+#include "params.h"
+#include <stdint.h>
 
-extern void __asm_ntt_SIMD_top(int *des, const int *table, const int *_constants);
-extern void __asm_ntt_SIMD_bot(int *des, const int *table, const int *_constants);
+extern void PQCLEAN_NAMESPACE_asm_ntt_SIMD_top(int *des, const int *table, const int *_constants);
+extern void PQCLEAN_NAMESPACE_asm_ntt_SIMD_bot(int *des, const int *table, const int *_constants);
 
-extern void __asm_intt_SIMD_top(int *des, const int *table, const int *_constants);
-extern void __asm_intt_SIMD_bot(int *des, const int *table, const int *_constants);
+extern void PQCLEAN_NAMESPACE_asm_intt_SIMD_top(int *des, const int *table, const int *_constants);
+extern void PQCLEAN_NAMESPACE_asm_intt_SIMD_bot(int *des, const int *table, const int *_constants);
 
 #define NTT(in) { \
-        __asm_ntt_SIMD_top(in, streamlined_CT_negacyclic_table_Q1_extended, constants); \
-        __asm_ntt_SIMD_bot(in, streamlined_CT_negacyclic_table_Q1_extended, constants); \
+        PQCLEAN_NAMESPACE_asm_ntt_SIMD_top(in, streamlined_CT_negacyclic_table_Q1_extended, constants); \
+        PQCLEAN_NAMESPACE_asm_ntt_SIMD_bot(in, streamlined_CT_negacyclic_table_Q1_extended, constants); \
     }
 
 #define iNTT(in) { \
-        __asm_intt_SIMD_bot(in, streamlined_inv_CT_table_Q1_extended, constants); \
-        __asm_intt_SIMD_top(in, streamlined_inv_CT_table_Q1_extended, constants); \
+        PQCLEAN_NAMESPACE_asm_intt_SIMD_bot(in, streamlined_inv_CT_table_Q1_extended, constants); \
+        PQCLEAN_NAMESPACE_asm_intt_SIMD_top(in, streamlined_inv_CT_table_Q1_extended, constants); \
     }
 
+#define ntt DILITHIUM_NAMESPACE(ntt)
 void ntt(int32_t a[ARRAY_N]);
+#define invntt_tomont DILITHIUM_NAMESPACE(invntt_tomont)
 void invntt_tomont(int32_t a[ARRAY_N]);
 
 static const int constants[16] = {

