--- work/dilithium3/scheme/rounding.c
+++ work-patched/dilithium3/scheme/rounding.c
@@ -109,18 +109,14 @@
 
     if (a0 > 0) {
         return (a1 + 1) & 15;
-    } else {
-        return (a1 - 1) & 15;
     }
-
+    return (a1 - 1) & 15;
     #elif GAMMA2 == (DILITHIUM_Q-1)/88
 
     if (a0 > 0) {
         return (a1 == 43) ?  0 : a1 + 1;
-    } else {
-        return (a1 ==  0) ? 43 : a1 - 1;
     }
-
+    return (a1 ==  0) ? 43 : a1 - 1;
     #endif
 
 }

