--- work/dilithium2/scheme/poly.c
+++ work-patched/dilithium2/scheme/poly.c
@@ -222,11 +222,11 @@
 *              - poly *a0: pointer to output polynomial with coefficients c0
 *              - const poly *a: pointer to input polynomial
 **************************************************/
-extern void __asm_poly_power2round(int32_t *, int32_t *, const int32_t *);
+extern void PQCLEAN_NAMESPACE_asm_poly_power2round(int32_t *, int32_t *, const int32_t *);
 void poly_power2round(poly *a1, poly *a0, const poly *a) {
     DBENCH_START();
 
-    __asm_poly_power2round(a1->coeffs, a0->coeffs, a->coeffs);
+    PQCLEAN_NAMESPACE_asm_poly_power2round(a1->coeffs, a0->coeffs, a->coeffs);
 
     DBENCH_STOP(*tround);
 }
@@ -437,6 +437,7 @@
         ctr1 += rej_uniform(a1->coeffs + ctr1, N - ctr1, buf1, buflen);
     }
 
+
 }
 
 /*************************************************
@@ -492,6 +493,7 @@
 
         #endif
 
+
     }
 
     DBENCH_STOP(*tsample);
@@ -780,11 +782,11 @@
 * Arguments:   - poly *r: pointer to output polynomial
 *              - const uint8_t *a: byte array with bit-packed polynomial
 **************************************************/
-extern void __asm_10_to_32(int32_t *, const uint8_t *);
+extern void PQCLEAN_NAMESPACE_asm_10_to_32(int32_t *, const uint8_t *);
 void polyt1_unpack(poly *r, const uint8_t *a) {
     DBENCH_START();
 
-    __asm_10_to_32(r->coeffs, a);
+    PQCLEAN_NAMESPACE_asm_10_to_32(r->coeffs, a);
 
     DBENCH_STOP(*tpack);
 }

