--- work/firesaber/scheme/__asm_mul.S
+++ work-patched/firesaber/scheme/__asm_mul.S
@@ -5,8 +5,8 @@
 .align 2
 .global __asm_asymmetric_mul
 .global ___asm_asymmetric_mul
-__asm_asymmetric_mul:
-___asm_asymmetric_mul:
+PQCLEAN_NAMESPACE__asm_asymmetric_mul:
+_PQCLEAN_NAMESPACE__asm_asymmetric_mul:
 
     push_all
 
@@ -91,8 +91,8 @@
 .align 2
 .global __asm_point_mul_extended
 .global ___asm_point_mul_extended
-__asm_point_mul_extended:
-___asm_point_mul_extended:
+PQCLEAN_NAMESPACE__asm_point_mul_extended:
+_PQCLEAN_NAMESPACE__asm_point_mul_extended:
 
     push_all
 

