--- work/firesaber/scheme/__asm_narrow.S
+++ work-patched/firesaber/scheme/__asm_narrow.S
@@ -4,8 +4,8 @@
 .align 2
 .global __asm_round
 .global ___asm_round
-__asm_round:
-___asm_round:
+PQCLEAN_NAMESPACE__asm_round:
+_PQCLEAN_NAMESPACE__asm_round:
 
 
     .equ srv, (SABER_EQ-SABER_EP)
@@ -50,8 +50,8 @@
 .align 2
 .global __asm_enc_add_msg
 .global ___asm_enc_add_msg
-__asm_enc_add_msg:
-___asm_enc_add_msg:
+PQCLEAN_NAMESPACE__asm_enc_add_msg:
+_PQCLEAN_NAMESPACE__asm_enc_add_msg:
 
     .equ srv, (SABER_EP-SABER_ET)
     .equ slv, (SABER_EP-1)
@@ -137,8 +137,8 @@
 .align 2
 .global __asm_dec_get_msg
 .global ___asm_dec_get_msg
-__asm_dec_get_msg:
-___asm_dec_get_msg:
+PQCLEAN_NAMESPACE__asm_dec_get_msg:
+_PQCLEAN_NAMESPACE__asm_dec_get_msg:
 
     .equ srv, (SABER_EP-1)
     .equ slv, (SABER_EP-SABER_ET)

