--- work/kyber512/scheme/neon_poly.c
+++ work-patched/kyber512/scheme/neon_poly.c
@@ -129,14 +129,14 @@
 *            - const poly *a: pointer to first input polynomial
 *            - const poly *b: pointer to second input polynomial
 **************************************************/
-extern void __asm_add_reduce(int16_t *, const int16_t *);
+extern void PQCLEAN_NAMESPACE_asm_add_reduce(int16_t *, const int16_t *);
 void neon_poly_add_reduce(int16_t c[KYBER_N], const int16_t a[KYBER_N]) {
-    __asm_add_reduce(c, a);
+    PQCLEAN_NAMESPACE_asm_add_reduce(c, a);
 }
 
-extern void __asm_add_add_reduce(int16_t *, const int16_t *, const int16_t *);
+extern void PQCLEAN_NAMESPACE_asm_add_add_reduce(int16_t *, const int16_t *, const int16_t *);
 void neon_poly_add_add_reduce(int16_t c[KYBER_N], const int16_t a[KYBER_N], const int16_t b[KYBER_N]) {
-    __asm_add_add_reduce(c, a, b);
+    PQCLEAN_NAMESPACE_asm_add_add_reduce(c, a, b);
 }
 
 /*************************************************
@@ -150,7 +150,7 @@
 *            - const poly *a: pointer to first input polynomial
 *            - const poly *b: pointer to second input polynomial
 **************************************************/
-extern void __asm_sub_reduce(int16_t *, const int16_t *);
+extern void PQCLEAN_NAMESPACE_asm_sub_reduce(int16_t *, const int16_t *);
 void neon_poly_sub_reduce(int16_t c[KYBER_N], const int16_t a[KYBER_N]) {
-    __asm_sub_reduce(c, a);
+    PQCLEAN_NAMESPACE_asm_sub_reduce(c, a);
 }

