--- work/kyber512/scheme/polyvec.c
+++ work-patched/kyber512/scheme/polyvec.c
@@ -19,7 +19,7 @@
 *                            (needs space for KYBER_POLYVECCOMPRESSEDBYTES)
 *              - const polyvec *a: pointer to input vector of polynomials
 **************************************************/
-void polyvec_compress(uint8_t r[KYBER_POLYVECCOMPRESSEDBYTES], const int16_t a[KYBER_K][KYBER_N]) {
+void polyvec_compress(uint8_t r[KYBER_POLYVECCOMPRESSEDBYTES], int16_t a[KYBER_K][KYBER_N]) {
     unsigned int i, j, k;
 
     #if (KYBER_POLYVECCOMPRESSEDBYTES == (KYBER_K * 352))
@@ -130,7 +130,7 @@
 *                            (needs space for KYBER_POLYVECBYTES)
 *              - const polyvec *a: pointer to input vector of polynomials
 **************************************************/
-void polyvec_tobytes(uint8_t r[KYBER_POLYVECBYTES], const int16_t a[KYBER_K][KYBER_N]) {
+void polyvec_tobytes(uint8_t r[KYBER_POLYVECBYTES], int16_t a[KYBER_K][KYBER_N]) {
     unsigned int i;
     for (i = 0; i < KYBER_K; i++) {
         poly_tobytes(r + i * KYBER_POLYBYTES, a[i]);

