--- work/kyber1024/scheme/__asm_base_mul.S
+++ work-patched/kyber1024/scheme/__asm_base_mul.S
@@ -1,16 +1,16 @@
 
-#include "macros.i"
+#include "macros.inc"
 
 #include "params.h"
 
 .align 2
-.global __asm_point_mul_extended
-.global ___asm_point_mul_extended
+.global PQCLEAN_NAMESPACE_asm_point_mul_extended
+.global _PQCLEAN_NAMESPACE_asm_point_mul_extended
 #ifndef __clang__
-  .type __asm_point_mul_extended, %function
+  .type PQCLEAN_NAMESPACE_asm_point_mul_extended, %function
 #endif
-__asm_point_mul_extended:
-___asm_point_mul_extended:
+PQCLEAN_NAMESPACE_asm_point_mul_extended:
+_PQCLEAN_NAMESPACE_asm_point_mul_extended:
 
     push_all
     Q         .req w20
@@ -71,13 +71,13 @@
 
 
 .align 2
-.global __asm_asymmetric_mul
-.global ___asm_asymmetric_mul
+.global PQCLEAN_NAMESPACE_asm_asymmetric_mul
+.global _PQCLEAN_NAMESPACE_asm_asymmetric_mul
 #ifndef __clang__
-.type __asm_asymmetric_mul, %function
+.type PQCLEAN_NAMESPACE_asm_asymmetric_mul, %function
 #endif
-__asm_asymmetric_mul:
-___asm_asymmetric_mul:
+PQCLEAN_NAMESPACE_asm_asymmetric_mul:
+_PQCLEAN_NAMESPACE_asm_asymmetric_mul:
 
     push_all
     Q         .req w28
@@ -226,13 +226,13 @@
 
 
 .align 2
-.global __asm_asymmetric_mul_montgomery
-.global ___asm_asymmetric_mul_montgomery
+.global PQCLEAN_NAMESPACE_asm_asymmetric_mul_montgomery
+.global _PQCLEAN_NAMESPACE_asm_asymmetric_mul_montgomery
 #ifndef __clang__
-.type __asm_asymmetric_mul_montgomery, %function
+.type PQCLEAN_NAMESPACE_asm_asymmetric_mul_montgomery, %function
 #endif
-__asm_asymmetric_mul_montgomery:
-___asm_asymmetric_mul_montgomery:
+PQCLEAN_NAMESPACE_asm_asymmetric_mul_montgomery:
+_PQCLEAN_NAMESPACE_asm_asymmetric_mul_montgomery:
 
     push_all
     Q         .req w28

