--- work/kyber1024/scheme/__asm_base_mul.S
+++ work-patched/kyber1024/scheme/__asm_base_mul.S
@@ -26,15 +26,15 @@
  * SOFTWARE.
  */
 
-#include "macros.i"
+#include "macros.inc"
 
 #include "params.h"
 
 .align 2
-.global __asm_point_mul_extended
-.global ___asm_point_mul_extended
-__asm_point_mul_extended:
-___asm_point_mul_extended:
+.global PQCLEAN_NAMESPACE__asm_point_mul_extended
+.global _PQCLEAN_NAMESPACE__asm_point_mul_extended
+PQCLEAN_NAMESPACE__asm_point_mul_extended:
+_PQCLEAN_NAMESPACE__asm_point_mul_extended:
 
     push_all
     Q         .req w20
@@ -246,10 +246,10 @@
 
 
 .align 2
-.global __asm_asymmetric_mul
-.global ___asm_asymmetric_mul
-__asm_asymmetric_mul:
-___asm_asymmetric_mul:
+.global PQCLEAN_NAMESPACE__asm_asymmetric_mul
+.global _PQCLEAN_NAMESPACE__asm_asymmetric_mul
+PQCLEAN_NAMESPACE__asm_asymmetric_mul:
+_PQCLEAN_NAMESPACE__asm_asymmetric_mul:
 
     push_all
     des       .req x11
@@ -595,10 +595,10 @@
 
 
 .align 2
-.global __asm_asymmetric_mul_montgomery
-.global ___asm_asymmetric_mul_montgomery
-__asm_asymmetric_mul_montgomery:
-___asm_asymmetric_mul_montgomery:
+.global PQCLEAN_NAMESPACE__asm_asymmetric_mul_montgomery
+.global _PQCLEAN_NAMESPACE__asm_asymmetric_mul_montgomery
+PQCLEAN_NAMESPACE__asm_asymmetric_mul_montgomery:
+_PQCLEAN_NAMESPACE__asm_asymmetric_mul_montgomery:
 
     push_all
     des       .req x11

