--- work/kyber768/scheme/neon_symmetric-shake.c
+++ work-patched/kyber768/scheme/neon_symmetric-shake.c
@@ -87,20 +87,18 @@
 void neon_kyber_shake256_prf(uint8_t *out1, uint8_t *out2,
                              size_t outlen,
                              const uint8_t key[KYBER_SYMBYTES],
-                             uint8_t nonce1, uint8_t nonce2)
-{
-  unsigned int i;
-  uint8_t extkey1[KYBER_SYMBYTES+1];
-  uint8_t extkey2[KYBER_SYMBYTES+1];
+                             uint8_t nonce1, uint8_t nonce2) {
+    unsigned int i;
+    uint8_t extkey1[KYBER_SYMBYTES + 1 + 15];
+    uint8_t extkey2[KYBER_SYMBYTES + 1 + 15];
 
-  for(i=0;i<KYBER_SYMBYTES;i++)
-  {
-    extkey1[i] = key[i];
-    extkey2[i] = key[i];
-  }
+    for (i = 0; i < KYBER_SYMBYTES; i++) {
+        extkey1[i] = key[i];
+        extkey2[i] = key[i];
+    }
 
-  extkey1[i] = nonce1;
-  extkey2[i] = nonce2;
+    extkey1[i] = nonce1;
+    extkey2[i] = nonce2;
 
-  shake256x2(out1, out2, outlen, extkey1, extkey2, sizeof(extkey1));
+    shake256x2(out1, out2, outlen, extkey1, extkey2, KYBER_SYMBYTES + 1);
 }

