--- work/kyber768/scheme/neon_polyvec.c
+++ work-patched/kyber768/scheme/neon_polyvec.c
@@ -83,7 +83,7 @@
 *            - const polyvec *a: pointer to first input vector of polynomials
 *            - const polyvec *b: pointer to second input vector of polynomials
 **************************************************/
-void neon_polyvec_add_reduce(int16_t c[KYBER_K][KYBER_N], const int16_t a[KYBER_K][KYBER_N]) {
+void neon_polyvec_add_reduce(int16_t c[KYBER_K][KYBER_N], int16_t a[KYBER_K][KYBER_N]) {
     unsigned int i;
     for (i = 0; i < KYBER_K; i++) {
         // c = c + a;
@@ -91,3 +91,4 @@
         neon_poly_add_reduce(c[i], a[i]);
     }
 }
+

