--- work/kyber768/scheme/neon_symmetric-shake.c
+++ work-patched/kyber768/scheme/neon_symmetric-shake.c
@@ -88,8 +88,8 @@
                              const uint8_t key[KYBER_SYMBYTES],
                              uint8_t nonce1, uint8_t nonce2) {
     unsigned int i;
-    uint8_t extkey1[KYBER_SYMBYTES + 1];
-    uint8_t extkey2[KYBER_SYMBYTES + 1];
+    uint8_t extkey1[KYBER_SYMBYTES + 1 + 15];
+    uint8_t extkey2[KYBER_SYMBYTES + 1 + 15];
 
     for (i = 0; i < KYBER_SYMBYTES; i++) {
         extkey1[i] = key[i];
@@ -99,5 +99,5 @@
     extkey1[i] = nonce1;
     extkey2[i] = nonce2;
 
-    shake256x2(out1, out2, outlen, extkey1, extkey2, sizeof(extkey1));
+    shake256x2(out1, out2, outlen, extkey1, extkey2, KYBER_SYMBYTES + 1);
 }

