--- work/kyber768/scheme/ntt.c
+++ work-patched/kyber768/scheme/ntt.c
@@ -1,8 +1,8 @@
-#include <arm_neon.h>
-#include "params.h"
+#include "NTT_params.h"
 #include "ntt.h"
+#include "params.h"
 #include "reduce.h"
-#include "NTT_params.h"
+#include <arm_neon.h>
 
 /*************************************************
 * Name:        ntt
@@ -13,7 +13,7 @@
 * Arguments:   - int16_t r[256]: pointer to input/output vector of elements of Zq
 **************************************************/
 // Merged NTT layer
-void ntt(int16_t r[256]){
+void ntt(int16_t r[256]) {
     NTT(r);
 }
 
@@ -24,11 +24,10 @@
 *              multiplication by Montgomery factor 2^16.
 *              Input is in bitreversed order, output is in standard order
 *
-* Arguments:   - int16_t r[256] in {-(q-1)/2,...,(q-1)/2} 
+* Arguments:   - int16_t r[256] in {-(q-1)/2,...,(q-1)/2}
 *              pointer to input/output vector of elements of Zq
 **************************************************/
-void invntt(int16_t r[256])
-{
-  iNTT(r);
+void invntt(int16_t r[256]) {
+    iNTT(r);
 
 }

