--- work/dilithium5/scheme/__asm_iNTT.S
+++ work-patched/dilithium5/scheme/__asm_iNTT.S
@@ -4,8 +4,8 @@
 .align 2
 .global __asm_intt_SIMD_top
 .global ___asm_intt_SIMD_top
-__asm_intt_SIMD_top:
-___asm_intt_SIMD_top:
+PQCLEAN_NAMESPACE__asm_intt_SIMD_top:
+_PQCLEAN_NAMESPACE__asm_intt_SIMD_top:
 
     push_all
     Q         .req w20
@@ -411,8 +411,8 @@
 .align 2
 .global __asm_intt_SIMD_bot
 .global ___asm_intt_SIMD_bot
-__asm_intt_SIMD_bot:
-___asm_intt_SIMD_bot:
+PQCLEAN_NAMESPACE__asm_intt_SIMD_bot:
+_PQCLEAN_NAMESPACE__asm_intt_SIMD_bot:
 
     push_all
     Q         .req w20

