--- work/dilithium5/scheme/__asm_NTT.S
+++ work-patched/dilithium5/scheme/__asm_NTT.S
@@ -4,8 +4,8 @@
 .align 2
 .global __asm_ntt_SIMD_top
 .global ___asm_ntt_SIMD_top
-__asm_ntt_SIMD_top:
-___asm_ntt_SIMD_top:
+PQCLEAN_NAMESPACE__asm_ntt_SIMD_top:
+_PQCLEAN_NAMESPACE__asm_ntt_SIMD_top:
 
     push_all
     Q         .req w20
@@ -182,8 +182,8 @@
 .align 2
 .global __asm_ntt_SIMD_bot
 .global ___asm_ntt_SIMD_bot
-__asm_ntt_SIMD_bot:
-___asm_ntt_SIMD_bot:
+PQCLEAN_NAMESPACE__asm_ntt_SIMD_bot:
+_PQCLEAN_NAMESPACE__asm_ntt_SIMD_bot:
 
     push_all
     Q         .req w20

