--- work/dilithium5/scheme/__asm_NTT.S
+++ work-patched/dilithium5/scheme/__asm_NTT.S
@@ -26,13 +26,13 @@
  * SOFTWARE.
  */
 
-#include "macros.i"
+#include "macros.inc"
 
 .align 2
-.global __asm_ntt_SIMD_top
-.global ___asm_ntt_SIMD_top
-__asm_ntt_SIMD_top:
-___asm_ntt_SIMD_top:
+.global PQCLEAN_NAMESPACE__asm_ntt_SIMD_top
+.global _PQCLEAN_NAMESPACE__asm_ntt_SIMD_top
+PQCLEAN_NAMESPACE__asm_ntt_SIMD_top:
+_PQCLEAN_NAMESPACE__asm_ntt_SIMD_top:
 
     push_simd
     Q         .req w8
@@ -163,10 +163,10 @@
     ret
 
 .align 2
-.global __asm_ntt_SIMD_bot
-.global ___asm_ntt_SIMD_bot
-__asm_ntt_SIMD_bot:
-___asm_ntt_SIMD_bot:
+.global PQCLEAN_NAMESPACE__asm_ntt_SIMD_bot
+.global _PQCLEAN_NAMESPACE__asm_ntt_SIMD_bot
+PQCLEAN_NAMESPACE__asm_ntt_SIMD_bot:
+_PQCLEAN_NAMESPACE__asm_ntt_SIMD_bot:
 
     push_simd
     Q         .req w8

