--- work/lightsaber/scheme/NTT_params.h
+++ work-patched/lightsaber/scheme/NTT_params.h
@@ -14,13 +14,13 @@
 #define invomegaQ1 8215090
 // R = 2^32 below
 // RmodQ1 = 2^32 mod^{+-} Q1
-#define RmodQ1 -929960
+#define RmodQ1 (-929960)
 // Q1prime = Q1^{-1} mod^{+-} 2^32
-#define Q1prime -155332095
+#define Q1prime (-155332095)
 // invNQ1 = NTT_N^{-1} mod Q1
 #define invNQ1 25171273
 // R2modQ1 = 2^32 mod^{+-} Q1
-#define R2modQ1 -929960
+#define R2modQ1 (-929960)
 // Q1prime2 = -Q1^{-1} mod^{+-} 2^32
 #define Q1prime2 155332095
 

